test/main.cpp

```diff
@@ -1,4 +1,36 @@
-#include "ostream"
+#include <ostream>
+
 
 #define CATCH_CONFIG_MAIN
 #include <catch.hpp>
+#include <hwlib.hpp>
+#include <qik_2s12v10.hpp>
+#include <test_usart.hpp>
+
+// Qik motorcontroller tests
+TEST_CASE("Get error tests", "[qik_2s12v10]") {
+    hwlib::pin_out_test reset_pin_test;
+    auto usart = r2d2::usart::test_usart_c<>();
+    // set default device id for get configuration in constructor of the qik
+    usart.add_receive_byte(10);
+    auto motor_controller =
+        r2d2::moving_platform::qik_2s12v10_c(usart, reset_pin_test);
+
+    uint8_t test_error_code;
```

Comment:
I think it is better to have a definition here. If no section has been crosses, this variable will cause undefined behaviour.

Reply:
The thing with sections in catch2 is a bit like templating, for every section there is a new catch 2 case, so it will always be set.

Reply:
Hmm, still not a fan of it. Will approve it since it is a minor issue